tool/teleport/common/usage.go

```diff
@@ -122,4 +122,7 @@ const (
 #  1. license.pem: Retrieve a license from your Teleport account https://teleport.sh
 #     if you are an Enterprise customer.
 #`
+
+	joinParamsUsage = `Comma-separated list of extra join parameters. Supported values:
```

Comment:
Let's not introduce a custom join params "CLI protocol" with custom parsing format, it will quickly get very messy.